gateway/ln-gateway/src/gateway_module_v2/events.rs

```diff
@@ -136,3 +137,97 @@ impl Event for CompleteLightningPaymentSucceeded {
 
     const KIND: EventKind = EventKind::from_static("complete-lightning-payment-succeeded");
 }
+
+/// Computes the `StructurePaymentEvents` for all LNv2 payments by filtering the
```

Comment:
Nit: Rustdoc comments supposed to be one liner header, and then description after line break

```
/// Computes the `StructurePaymentEvents` for all LNv2 payments
///
/// Filters the ...
```

Yes, it kind of makes a difference in the UX e.g. when using LSPs and looking up what a function does.

Reply:
Done